### DISCUSSION TITLE
Not issue! Asking just a question..

### DISCUSSION THREAD

**Discussion by Saikatsaha1996** [perpetualKid/FreeTrainSimulator#120] [General]
Any `Mono` comfortable Openrail available?

**Comment by perpetualKid** [comment]
while this particular version of OpenRails is based on .NET (Core), which is available on multiple platforms including Windows and Linux, however there is no OpenRails version which does run natively on Linux (or other OS than Windows). For guidance to run this version on Linux, please see this article https://github.com/perpetualKid/ORTS-MG/wiki/Linux-Wine

I'm not sure if there is any OpenRails version at all which runs on Mono platform, to be frank, I can't see a need for that. On Windows, appropriate .NET versions are available, other platforms (including Linux) will need more work in various parts to make them natively available, so the choice of a platform such as Mono would just add additional migration overhead, with no particular advantage.

Maybe you could give some more context what you are trying to achieve, to provide better guidance @Saikatsaha1996

**Reply by Saikatsaha1996** [reply]
> while this particular version of OpenRails is based on .NET (Core), which is available on multiple platforms including Windows and Linux, however there is no OpenRails version which does run natively on Linux (or other OS than Windows). For guidance to run this version on Linux, please see this article https://github.com/perpetualKid/ORTS-MG/wiki/Linux-Wine
> 
> I'm not sure if there is any OpenRails version at all which runs on Mono platform, to be frank, I can't see a need for that. On Windows, appropriate .NET versions are available, other platforms (including Linux) will need more work in various parts to make them natively available, so the choice of a platform such as Mono would just add additional migration overhead, with no particular advantage.
> 
> Maybe you could give some more context what you are trying to achieve, to provide better guidance @Saikatsaha1996

Thank you for your reply..
I already run successfully(Openrail 1.2) in wine with arm64 architecture in Android..
#105 this was Android device..

But it was proot linux..
But now i want to run in pure Android with `mono`
That's why I was asked..
Thank you for your response 😊

**Reply by Saikatsaha1996** [reply]
> while this particular version of OpenRails is based on .NET (Core), which is available on multiple platforms including Windows and Linux, however there is no OpenRails version which does run natively on Linux (or other OS than Windows). For guidance to run this version on Linux, please see this article https://github.com/perpetualKid/ORTS-MG/wiki/Linux-Wine
> 
> I'm not sure if there is any OpenRails version at all which runs on Mono platform, to be frank, I can't see a need for that. On Windows, appropriate .NET versions are available, other platforms (including Linux) will need more work in various parts to make them natively available, so the choice of a platform such as Mono would just add additional migration overhead, with no particular advantage.
> 
> Maybe you could give some more context what you are trying to achieve, to provide better guidance @Saikatsaha1996

If it possible please disable or delete the post!

**Comment by Saikatsaha1996** [comment]
Hello Sir! I am playing openrail in Arm64 device via emulator.. I received best performance in box64.. but with box64 unable to run openrail most probably directX issue..  I unable to understand how can I solve this issue.. if you help for this it would be great..
![Screenshot_2024-01-03-00-45-21-03_b6022cb7d1a61f21e4fe914c8728a54c](https://github.com/perpetualKid/ORTS-MG/assets/72664192/ef43ba30-a7d2-4c2c-9987-631662466f77)


I successfully played openrail in another emulator but this emulator only for 32 bit application.. so I want to play Openrail with 64bit support box64..
![Screenshot_2024-01-02-22-44-16-64_a0b0d0b5d43383e387863816414c311f](https://github.com/perpetualKid/ORTS-MG/assets/72664192/0f383538-320e-4fed-9372-0cadb863368f)
![Screenshot_2024-01-02-22-41-58-98_a0b0d0b5d43383e387863816414c311f](https://github.com/perpetualKid/ORTS-MG/assets/72664192/16c2a372-027b-4ad1-857a-319ffe0de05c)
![Screenshot_2024-01-02-22-28-27-13_a0b0d0b5d43383e387863816414c311f](https://github.com/perpetualKid/ORTS-MG/assets/72664192/e3af6719-3484-4d36-9a44-846021fa3640)

**Reply by perpetualKid** [reply]
seems to be an issue while loading a texture, but without the full error message or the log file, the provided information doesn't give any clue.

**Reply by Saikatsaha1996** [reply]
How can I share .log file? I mean in which folder log file will be store? Than I can share log..

**Reply by perpetualKid** [reply]
in Windows it*s stored on the Desktop, not sure how that translated in an emulator
